prototype2/CMakeLists.txt

```diff
@@ -52,8 +52,14 @@ add_subdirectory(udp)
 add_subdirectory(adc_readout)
 
 add_custom_target(runefu
-    COMMAND efu "-d" "../modules/mgmesytec" "-s" "1"
-    DEPENDS efu mgmesytec
+    COMMAND efu "-d" "../modules/mgmesytec" "-s" "1" "--dumptofile" "deleteme_"
+    COMMAND efu "-d" "../modules/mgcncs2" "-s" "1" "--dumptofile" "deleteme_"
+    COMMAND efu "-d" "../modules/sonde" "-s" "1" "--dumptofile" "deleteme_"
+    COMMAND efu "-d" "../modules/mbcaen" "-s" "1" "--dumptofile" "deleteme_"
+    COMMAND efu "-d" "../modules/AdcReadout" "-s" "1"
+    COMMAND efu "-d" "../modules/udp" "-s" "1"
+    COMMAND efu "-d" "../modules/gdgem" "-f" "nocfgfile" "-s" "2"
```

Comment:
Interesting that you actually run every module. Is the outcome of running __make runefu__ a part of continuous integration?

Reply:
Yes, you could say that. It makes sure that the efu can load the detector and not crash,  or check if new mandatory options are introduced. It also has the side effect of increasing 
test coverage although this was not the driving force for adding this.

Reply:
This is a kind of system test, so I would question whether it should contribute to coverage? I.e. should we report coverage only of code that is tested with targeted unit tests? You could do this by doing the following:
* run unit tests
* generate coverage
* run these system tests

I have been debating whether I should do this in daquiri, as it also "overreports" coverage because of systems tests. I disable the system tests when testing locally so I have a more honest view. But this is also not wrong. Up to you.